pycheribuild/projects/cross/webkit.py

```diff
@@ -0,0 +1,128 @@
+#
```

Comment:
This looks pretty much identical to the other version that I wrote, maybe add a 
Co-authored-by: Alex Richardson <Alexander.Richardson@cl.cam.ac.uk>` line to the commit message.

Reply:
In that case can we make them share code?

Reply:
Sorry, I don't mean the existing the qtwebkit implementation, I meant the externally-managed private version that existed prior to open-sourcing the new webkit repo.